### PR TITLE
Cms migration schema

### PR DESCRIPTION
**Pull request by rjweeks**
### HOCS-5147

This is an initial draft of the CMS migration message schema.

It is subject to significant change during development team discussions. I will setup a meeting for this discussion.

### Testing 

Schema validates against Schema Draft 2020-12

### CODE REVIEW

**Review by LiamMacP — APPROVED** [review]
I don't know enough about this schema to know if this is right for the moment. But it is indeed a schema - I've added one question as it would be good to use the latest if we can.

File: src/main/resources/cms-mig.json [hunk 1]
```diff
@@ -0,0 +1,128 @@
+{
+  "scehma": "https://json-schema.org/draft/2019-09/schema",
```
Comment (robertdeniszczyc2) [inline src/main/resources/cms-mig.json:2]:
```suggestion
  "schema": "https://json-schema.org/draft/2019-09/schema",
```

File: src/main/resources/cms-mig.json [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  "title": "CMS DECS Migration",
```
Comment (robertdeniszczyc2) [inline src/main/resources/cms-mig.json:3]: We should also specify a version which the schema is using, such as swagger: "2.0" and those that match openapi: 3.0.n (for example, openapi: 3.0.0).

Also means then you can view the models using an openapi/swagger viewer like https://editor.swagger.io/

**Review by robertdeniszczyc2 — CHANGES_REQUESTED** [review]

File: src/main/resources/cms-mig.json (hunk 1 above)
Comment (LiamMacP) [reply src/main/resources/cms-mig.json:2]: Good catch this!

[the remark this answers is not in the document]
Comment (paul-mc) [reply src/main/resources/cms-mig.json:2]: yeah, good spot - I don't see any reason why we shouldn't use the latest `https://json-schema.org/draft/2020-12/schema`

[the remark this answers is not in the document]
Comment (rjweeks) [reply src/main/resources/cms-mig.json]: Updated the schema version.

**Comment by rjweeks** [comment]
Closed. The schema has been moved to its own repository hocs-migration-schema

**Pull request closed without merging**